kiwi/storage/loop_device.py

```diff
@@ -98,12 +101,13 @@ def create(self, overwrite: bool = True):
         )
         self.node_name = loop_call.output.rstrip(os.linesep)
```

Comment:
Would it make sense to move `create` into `__init__` or `__enter__`?

From my PoV there's little use for a `LoopDevice` that is not created. However, I am unsure whether this would be a good API or whether that is considered good python style

Reply:
That part I'd like to keep for now. Reason is mostly consistency. All Device classes works in the way that you can create the object instance and it does not immediately run the device magic but there is a method to initiate it. For the loop device class it's true there is not much benefit but other device classes provides methods that could be called prior the creation/initialization code. As a followup refactoring we can look again into this topic but for the move from finalizers to context manager I think the basic class behavior should not change